modules/ROOT/pages/revisions.adoc

```diff
@@ -175,18 +175,20 @@ Whenever a document is accessed its revision tree (or at least some portion of i
 === Control
 You can control the size of the revision cache using the {configuration-schema-database--xref--cache--rev-cache} settings within the configuration file, specifically:
 
-* {configuration-schema-database--xref--cache--rev-cache--size}
+* xref:configuration-schema-database.adoc#cache-rev_cache-max_memory_count_mb[rev_cache.max_memory_count_mb]
+
 * {configuration-schema-database--xref--cache--rev-cache--shard-count}
 
+* {configuration-schema-database--xref--cache--rev-cache--size}
 
-[#lbl-size]
-=== Size
-Use the {configuration-schema-database--xref--cache--rev-cache--size} setting to specify the total number of document revisions to be cached in memory for all (recently accessed) documents.
+[#lbl-max-memory]
```

Comment:
@ElliotFrancisHunter this config should be colocated with the other rev cache size config, size. Shouldn't have sharding which is unrelated in between them.

These two configs work hand in hand as it is very well explained in the db config page: 

> When set this memory limit will work in in hand with revision cache size parameter. So you will potentially get eviction at revision cache both based off memory footprint and number of items in the cache. 

We should communicate to users what the connection between those two configs is (controlling size of rev cache) and the implications for setting both that is outlined in the quote above. Maybe even have them colocated in a subsection. Thoughts?

Reply:
Done